state/in-memory/in-memory.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package inmemory
+
+import (
+	"fmt"
+
+	"github.com/dgraph-io/ristretto"
+
+	"github.com/dapr/kit/logger"
+
+	"github.com/dapr/components-contrib/state"
+)
+
+// StateStore is an in-memory state store.
+// The first version is used for perf test.
```

Comment:
Does it require us to call this out explicitly in our docs so that no one ends up using it?